semgrep/semgrep/config_resolver.py

```diff
@@ -348,6 +407,42 @@ def _rename_rule_ids(valid_configs: Mapping[str, Sequence[Rule]]) -> None:
                     f"{Config._convert_config_id_to_prefix(config_id)}{rule.id or MISSING_RULE_ID}"
                 )
 
+    @staticmethod
```

Comment:
For ease of future maintainability, can we put these two functions in a separate file?